manifests/init.pp

```diff
@@ -121,8 +121,9 @@
 # @author Lucas Yamanishi <lucas.yamanishi@onyxpoint.com>
 #
 class simp_grafana (
-  Simplib::Netlist              $trusted_nets            = $::simp_grafana::params::trusted_nets,
-  Boolean                       $firewall                = $::simp_grafana::params::firewall,
+  Simplib::Netlist              $trusted_nets            = simplib::lookup('simp_options::trusted_nets', { 'default_value' => ['127.0.0.0/8'] }),
```

Comment:
This should be nailed to `127.0.0.1` by default. In the new containerized world, not even the home block is safe any more.